src/main/scala-2.13/reactor/core/scala/VersionedScannable.scala

```diff
@@ -0,0 +1,27 @@
+package reactor.core.scala
+
+import scala.jdk.CollectionConverters._
+
+trait VersionedScannable { self: Scannable =>
+  def actuals(): LazyList[_ <: Scannable] = jScannable.actuals().iterator().asScala.map(js => js: Scannable).to(LazyList)
+
+  def inners(): LazyList[_ <: Scannable] = jScannable.inners().iterator().asScala.map(js => js: Scannable).to(LazyList)
+
+  /**
+    * Return a [[Stream]] navigating the [[org.reactivestreams.Subscription]]
```

Comment:
Ought to be `[[LazyList]]` in these 2.13 docs

Reply:
Thank you... I've updated it